pkg/apis/pmemcsi/v1alpha1/deployment_types.go

```diff
@@ -81,6 +81,8 @@ type DeploymentStatus struct {
 
 	// Phase indicates the state of the deployment
 	Phase DeploymentPhase `json:"phase,omitempty"`
+	// LastUpdated time of the deployment status
+	LastUpdated metav1.Time `json:"lastUpdated,omitempty"`
```

Comment:
Long term we should reconsider what we put into our status and align with https://github.com/kubernetes/enhancements/pull/1624.

Instead of relying on time, `observedGeneration` (mentioned in that KEP discussion without further explanation) may be better because it is immune to time shifts.

Okay for now.